portal/templates/redesign/help-and-support_new.html

```diff
@@ -0,0 +1,87 @@
+{% extends 'redesign/base_new.html' %}
+{% load staticfiles %}
+{% load app_tags %}
+
+{% block css %}
+{{ block.super }}
+{% endblock css %}
+
+{% block scripts %}
+{{ block.super }}
+{% endblock scripts %}
+
+{% block pageID %}id='homeWrapper'{% endblock %}
+
+{% block content %}
+{{ block.super }}
+
+<div class="banner banner--teacher row">
+    <div class="col-sm-5 col-sm-offset-2">
+        <h1 class="banner__text--primary">Help and support</h1>
+        <p class="body-text">If you&rsquo;ve run into problems, please look at our <a href="#faqs">FAQs</a> below for advice to commonly asked questions. If you still can&rsquo;t
+            find the answers, please <a href="#contact">contact us</a>. One of the Code for Life team will get back to you and resolve your issues.</p>
+    </div>
+    <div class="col-sm-5">
+        <div class="hexagon"><div class="hexagon-in1"><div class="hexagon-in2 hexagon-in2--about"></div></div></div>
+    </div>
+</div>
+
+<div class="background col-sm-8 col-center">
+    <h3>Browser support</h3><br>
+    <p class="body-text">The Code for Life website and the Rapid Router app should be supported by all modern browsers. Please note this means that Internet Explorer 8
+        and Internet Explorer 9 are not supported. Please upgrade to Internet Explorer 10 or above, or use an alternative browser if you are currently using these
+        browsers.</p><br>
+    <p class="body-text">Should you experience any difficulties with support for browsers or other devices you are using which you are unable to resolve, please
+        <a href="#contact">contact us</a> for assistance.</p>
+</div>
+<div id="faqs">
+    <div class="background col-sm-8 col-center">
+        <h3>FAQs</h3><br>
+        <p class="body-text"><b>+ I am a school student and have forgotten my password. How do I reset it?</b></p>
```

Comment:
Bold is deprecated now in favour of Emphasis.  However, think carefully about whether you want to emphasise these questions (eg to search engines and screen readers), or whether you just want them to visually look a certain way.  If it's the latter...  well, you know.